src/core/manager.c

```diff
@@ -2699,7 +2676,7 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                         if (strv_extend(&m->deserialized_subscribed, l+11) < 0)
                                 log_oom();
 
-                } else
+                } else if (!startswith(l, "kdbus-fd=")) /* ignore this one */
```

Comment:
@keszybz , oh, right.
https://github.com/systemd/systemd/blob/83f12b27d14853e7c89a326f7cd31a6c739d378e/src/core/main.c#L1813
```c
        /* This will close all file descriptors that were opened, but
         * not claimed by any unit. */
        fds = fdset_free(fds);
```